packages/framework/pr_tools/trilinosprhelpers/unittests/environment-specs.ini

```diff
@@ -180,15 +180,6 @@ envvar-find-in-path OMPI_CXX : icpc
 envvar-find-in-path OMPI_CC  : icc
 envvar-find-in-path OMPI_FC  : ifort
 
-
-[SEMS_PRE]
-use MODULE-PURGE
-module-load sems-cmake
-module-load sems-ninja
-
 [rhel8_sems-gnu-openmpi]
-use SEMS_PRE
-module-load sems-gcc
 use MPI-COMPILER-VARS
-module-load sems-openmpi
 use OMPI-GNU-VARS
```

Comment:
Do you need this part of the change any more?

Reply:
Good point, I tested with the removal of the rest of the environment-specs sections except an empty `[rhel8_sems-gnu-openmpi]`